src/libsystemd/sd-netlink/netlink-message.c

```diff
@@ -859,9 +859,9 @@ static int netlink_container_parse(sd_netlink_message *m,
 int sd_netlink_message_enter_container(sd_netlink_message *m, unsigned short type_id) {
         const NLType *nl_type;
         const NLTypeSystem *type_system;
+        unsigned short size;
         void *container;
         uint16_t type;
-        size_t size;
```

Comment:
similar here: stick with size_t, only size_t can accurately describe sizes in memory, appropriate for all archs